Research/ResearchWarcrimes.md

```diff
@@ -0,0 +1,39 @@
+<!-- It's not necessary to follow this format, as long as you provide a coherent and structured document -->
+
+## Abstract
+
+Research should be a department side project of the USCM capable of providing buffs to marines, debuffs/buffs to xenos and unique mechanics such as corrupted xenomorphs and being able to reverse engineer Predator technology to provide something in return.
+ First act of rejuvenation is including new features that researchers can do and provide more co-operation between OT and the marines.
+
+## Goals
+
+1. To provide multiple methods of assisting the USCM be it with buffs, debuffs, unique mechanics etc.
+
+2. To make said methods have each of their own difficulty on making them.
+
+3. To make said methods be able to be used in combinations together.
+
+4. To encourage co-operation between other departments and the marines.
+
+## Non-goals
+
+1. Significantly affect balance over a certain time period. This design does not aim to change the balance of the round in a quick manner, said methods will always take time to create, giving the antagonists time to not worry about it.
+
+2. Make the USCM dependant on it. Research is intended to be a side project just like OT and is going to treated as a side mini goal for the USCM.
+
+3. Make said features easily available. Due to their power factor, these features/methods should have a certain rarity to them to avoid them affecting the rounds too much.
+
+## Content
+
+The features added would be, a new way that makes chemicals work by applying it differently via TOUCH (through spraying and gas) and making said chemicals now also work on xenomorphs (depending if the property can do it). 
+New additional properties that also contain the new features of TOUCH application for more variety.
+New code of chemicals to use the new mechanics of properties (example, Epinephrine's unique intravenous feature has now been replaced by a new property that does the same thing but now it can be added to other chemicals with the help of the new property) 
+Tweak the cost of properties based on the new features to make them less/more available (Stimulant property costs increased while underused properties such as organ healing ones decreased) 
```

Comment:
Properties gained from items/bodies etc. ?
There is a lot of general information here but little details on direct implementation method, and for research one will have to be a bit more precise.

Reply:
Question anwsered